### PR TITLE
Added the 2020.9 release notes + gifs for the doc in the img folder

### PR DESCRIPTION
**Pull request by liam-philiben**
2020.9 Release Notes + assets

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -3,6 +3,60 @@ id: ReleaseNotes
...
+
+#### Post-Processing: Bloom
+
+Have you always wanted to add a satisfying glow to your experiences? Or make your scene look like something out of Tron or a video game? Well, you’re in luck! 
+
+New to Enklu Cloud this release is our first in-editor, post-processing shader: **Bloom**! You can access Bloom settings in your project by navigating to the Scene Settings tab, on the left sidebar of the editor.  
+
+For more information on Bloom, check out our official guide [here](WebEditorBasics/PostProcessing).
+
+<img src="https://cdn.enklu.com/web-assets/bloom.gif" width="100%" alt="Bloom"/>
```
Comment (jsundra) [inline docs/ReleaseNotes.md:18]: This image should be added with the `![alt-text][path]` syntax, like `Bloom`. The underlying doc framework will handle creating all of the tags/style when it generates the md files into html.

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -3,6 +3,60 @@ id: ReleaseNotes
...
+
+### Improvements
+
+#### Scripting UI/UX
+
+Finding and editing a script in Enklu Cloud has never been easier with our improved Scripting UI! There’s now a dedicated tab for your Scripts right next to your Assets.
+
+<img src="https://cdn.enklu.com/web-assets/script_tab.gif" width="100%" alt="Script Tab"/>
```
Comment (jsundra) [inline docs/ReleaseNotes.md:26]: ditto here... and below as well :D

File: docs/ReleaseNotes.md [hunk 3]
```diff
@@ -136,6 +200,8 @@ interpolation.
 
 - Fixed error where calling undo resulted in an error
 
+***
```
Comment (jsundra) [inline docs/ReleaseNotes.md:203]: Maybe for 2020.10 we'll split the release notes into sub-pages so this becomes more manageable :D

**Review by jsundra — CHANGES_REQUESTED** [review]
Looks pretty good overall, only comments are about the `` tags.

The image file: static/img/product/releases/2020.9/script_edit.gif is 30.7mb! We might have forgotten to mention, so apologies, but in general it's great to keep gifs under 5mb if possible. In a pinch we'll grumble and let things go to 10mb if the quality otherwise is awful. Let us know if you have any trouble with it, wrangling gifs and dealing with encoding and quality and framerate is very much so a skill learned through much pain.

**Review by gdamron — CHANGES_REQUESTED** [review]
Looks good. Just one request: can you get the size of `script_edit.gif` down a bit? you could probably remove every other frame and chop a little off the beginning and end.

File: docs/ReleaseNotes.md (hunk 3 above)
Comment (gdamron) [reply docs/ReleaseNotes.md:203]: yeah, that way we can continue to be liberal with our use of gifs

File: docs/ReleaseNotes.md (hunk 1 above)
Comment (ShannonKalei) [reply docs/ReleaseNotes.md:18]: The way you've added the images to this repo, you'll have to use that markdown syntax James is talking about.  Though the other images for the release notes Grant did not add in this same way, that's why they are using the img tags.  Grant is too cool for us.  Using the markdown syntax is fine for now, though eventually we may want to swap everything over to Grant's coolness 😎

**Review by ShannonKalei — COMMENTED** [review]
When you use the markdown syntax for your images, you should be able to see them locally using `npm start` so you can know you did it right 😉

**Comment by gdamron** [comment]
> When you use the markdown syntax for your images, you should be able to see them locally using `npm start` so you can know you did it right 😉

Oh, ha, nice catch. Yeah, I probably had some good reason for pushing the gifs to our cdn (🙂), but better to do what Shannon suggested.

**Comment by liam-philiben** [comment]
thanks for the feedback everyone! I was wondering why the images weren't loading when I was formatting the doc with the local server preview yesterday, now I know why haha

**Comment by liam-philiben** [comment]
okay, just updated the image tags and made the script edit gif a lot smaller

**Review by gdamron — APPROVED** [review]
👍 welcome to the codebase

**Review by jsundra — APPROVED** [review]
image

**Pull request merged**